pkg/apiserver/profile/router.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package profile
+
+import (
+	"sync"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/pingcap-incubator/tidb-dashboard/pkg/config"
+	"github.com/pingcap-incubator/tidb-dashboard/pkg/dbstore"
+)
+
+// Service is used to provide a kind of feature.
+type Service struct {
+	config *config.Config
+	db     *dbstore.DB
+	tasks  sync.Map
+}
+
+// NewService creates a new service.
+func NewService(config *config.Config, db *dbstore.DB) *Service {
+	autoMigrate(db)
+	return &Service{config: config, db: db, tasks: sync.Map{}}
+}
+
+// Register register the handlers to the service.
```

Comment:
Looks like you can put router together with the handler

Reply:
What's the benefit?

Reply:
So that route path is not maintained in two separated files..